src/helpers/copy.js

```diff
@@ -10,12 +10,6 @@
  * governing permissions and limitations under the License.
  */
 
-/**
- * Creates the copy source/dest object from the current contexts
- * @param {Request} req the Request object
- * @param {Object} daCtx the DA Context
- * @return {Promise<{destination: string, source: string}|{}>}
- */
 export default async function copyHelper(req, daCtx) {
```

Comment:
Why is the jsdoc removed? Was it incorrect?

Reply:
Didn't mean to remove it. I put it back.